File: java/link-message-client/src/main/java/link/message/client/messager/PersonMessageReceiver.java
package link.message.client.messager;

/**
 * 个人消息接受者
 * @author zhongt
 *
 */
public class PersonMessageReceiver extends MessageReceiver {
	
	public PersonMessageReceiver() {
		
	}
	
	public PersonMessageReceiver(String id, String name) {
		this.toId   = id;
	    this.toName = name;
	}

	@Override
	public String getToType() {
		return MessageSendOrReceiverType.PERSON;
	}

	@Override
	public void setToType(String toType) {
		super.setToType(MessageSendOrReceiverType.PERSON);
	}
}

File: net/Link.Message.Client/messager/PersonMessageReceiver.cs
namespace Link.Message.Client.messager
{

	/// <summary>
    /// 个人消息接受者
	/// </summary>
	public class PersonMessageReceiver : MessageReceiver
	{

		public PersonMessageReceiver()
		{

		}

		public PersonMessageReceiver(string id, string name)
		{
			ToId = id;
			ToName = name;
		}

		public override MessageSendOrReceiverType ToType
		{
			get
			{
				return MessageSendOrReceiverType.Person;
			}
			set
			{
				base.ToType = MessageSendOrReceiverType.Person;
			}
		}

	}
}